scripts/create_baseline_stubs.py

```diff
@@ -10,11 +10,16 @@
 
 from __future__ import annotations
 
+import aiohttp
 import argparse
+import asyncio
 import os
 import re
 import subprocess
 import sys
+import urllib.parse
+
+from stubsabot import fetch_pypi_info, get_github_api_headers
```

Comment:
Feels a bit weird importing from stubsabot here -- the other way of doing this would be to create a third module that they both import these functions from

Reply:
After sleeping on it, I realised that importing from stubsabot here only actually saved us about 5 lines of code. It was unnecessary.